tests/sheet/sheet_manipulation_plugin.test.ts

```diff
@@ -1188,6 +1233,45 @@ describe("Rows", () => {
       expect(getCellsObject(model, "sheet1")).toMatchSnapshot();
       expect(getCellsObject(model, "sheet2")).toMatchSnapshot();
     });
+
+    test("On move", () => {
+      model = new Model({
+        sheets: [
+          {
+            id: "sheet1",
+            colNumber: 7,
+            rowNumber: 7,
+            cells: {
+              A1: { content: "R1" },
+              B1: { content: "1" },
+              C1: { content: "=B1" },
+              D1: { content: "=B1+C1" },
+              A2: { content: "R2" },
+              B2: { content: "2" },
+              C2: { content: "=B2" },
+              D2: { content: "=B2+C2" },
+              A3: { content: "R3" },
+              B3: { content: "3" },
+              C3: { content: "=B3" },
+              D3: { content: "=B3+C3" },
+              A4: { content: "R4" },
+              B4: { content: "=B1+C1" },
+              C4: { content: "=B2+C2" },
+              D4: { content: "=B3+C3" },
+            },
+          },
+        ],
+      });
+      moveRows(model, 2, [0], "after");
+      expect(getCellsObject(model, "sheet1")).toMatchSnapshot();
```

Comment:
why is this test using snapshot while the previous one is not ? Just don't use snapshot tests when not needed
- it makes it harder to read the test. I have no clue if the references were correctly moved reading this
- it makes us 10x more likely to ignore a bug, and just update the snapshots